Moggles.Domain/Application.cs

```diff
@@ -46,14 +46,14 @@ public static Application Create(string appName, string defaultEnvironmentName,
             return app;
         }
 
-        public void AddFeatureToggle(string toggleName, string notes, bool isPermanent = false)
+        public void AddFeatureToggle(string toggleName, string notes, bool isPermanent = false, string workItemIdentifier="")
```

Comment:
Why does _workItemIdentifier_ have a default value of empty string? Why does it need a default value set?